src/components/ExchangePage/logos/coinhouse.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import React from 'react'
+
+export default ({ width }: { width: number }) => (
```

Comment:
make it pure or external const (can wrap a `<g>`) ;)

Reply:
or use fragment eheh